images/win/scripts/Installers/Validate-AWS-CLI-SM.ps1

```diff
@@ -0,0 +1,14 @@
+################################################################################
+##  File:  Validate-AWS-CLI-SM.ps1
```

Comment:
Do we really need the separate script for validation if we install in the single script?

Reply:
Done.